source/manual/access-ci.html.md

```diff
@@ -4,16 +4,15 @@ title: Get access to CI (Jenkins)
 section: Testing
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2017-07-12
+last_reviewed_on: 2017-10-04
 review_in: 2 months # We may soon be moving to public GitHub for authentication.
 ---
 
 Our [Jenkins CI environment][ci] uses GitHub Enterprise for authorisation. You need to be given access first, otherwise you won't see any projects, or see a message that you're unauthorised.
```

Comment:
We use public GitHub now, right @surminus?

Reply:
A user just needs to be added to the GOV.UK team in Github for CI.